spec/rubocop/config_spec.rb

```diff
@@ -828,12 +834,171 @@ def cop_enabled(cop_class)
+                      actionmailer (= 4.1.0)
+                      actionpack (= 4.1.0)
+
+                  PLATFORMS
+                    ruby
+
+                  DEPENDENCIES
+                    ruby-extensions (~> 1.9.0)
+
+                  RUBY VERSION
+                     ruby 2.0.0p0
+
+                  BUNDLED WITH
+                    1.16.1
+                HEREDOC
+              create_file(lock_file_path, content)
+              expect(configuration.target_ruby_version).to eq 2.0
+            end
+
+            it 'uses MRI Ruby version when it has multiple digits' do
+              content =
+                <<-HEREDOC
+                  GEM
+                    remote: https://rubygems.org/
+                    specs:
+                      actionmailer (4.1.0)
+                      actionpack (= 4.1.0)
+                    rails (4.1.0)
+                      actionmailer (= 4.1.0)
+                      actionpack (= 4.1.0)
+
+                  PLATFORMS
+                    ruby
+
+                  DEPENDENCIES
+                    ruby-extensions (~> 1.9.0)
+
+                  RUBY VERSION
+                     ruby 20.10.100p450
+
+                  BUNDLED WITH
+                    1.16.1
+                HEREDOC
+              create_file(lock_file_path, content)
+              expect(configuration.target_ruby_version).to eq 20.10
+            end
+
+            it "uses the default Ruby when Ruby is not in #{file_name}" do
+              content =
+                <<-HEREDOC
+                  GEM
+                    remote: https://rubygems.org/
+                    specs:
+                      addressable (2.5.2)
+                        public_suffix (>= 2.0.2, < 4.0)
+                      ast (2.4.0)
+                      bump (0.5.4)
+
+                  PLATFORMS
+                    ruby
+
+                  DEPENDENCIES
+                    bump
+                    bundler (~> 1.3)
+                    ruby-extensions (~> 1.9.0)
+
+                  BUNDLED WITH
+                    1.16.1
+                HEREDOC
+              create_file(lock_file_path, content)
+              default = RuboCop::Config::DEFAULT_RUBY_VERSION
+              expect(configuration.target_ruby_version).to eq default
+            end
+
+            it "uses the default Ruby when jruby is in #{file_name}" do
+              content =
+                <<-HEREDOC
+                  GEM
+                    remote: https://rubygems.org/
+                    specs:
+                      addressable (2.5.2)
+                        public_suffix (>= 2.0.2, < 4.0)
+                      ast (2.4.0)
+                      bump (0.5.4)
+
+                  PLATFORMS
+                    ruby
+
+                  DEPENDENCIES
+                    bump
+                    bundler (~> 1.3)
+                    ruby-extensions (~> 1.9.0)
+
+                  RUBY VERSION
+                     ruby 2.0.0p0 (rbx 3.42)
```

Comment:
The example's description is "uses the default Ruby when **jruby**", but the lockfile has `rbx 3.42`. Should it be "jruby"?